src/qt/bitcoinunits.cpp

```diff
@@ -40,9 +40,9 @@ QString BitcoinUnits::name(int unit)
 {
     switch(unit)
     {
-    case BTC: return QString("BTC");
-    case mBTC: return QString("mBTC");
-    case uBTC: return QString::fromUtf8("μBTC");
+    case BTC: return QString(CURRENCY_UNIT.c_str());
+    case mBTC: return QString(("m" + CURRENCY_UNIT).c_str());
```

Comment:
Did you test this? const char * + const char * is going to be a pointer to some undefined memory...